### PR TITLE
project_axion add fonts

### PR DESCRIPTION
**Pull request by SergeyN83**

### CODE REVIEW

**Comment by KsuBurn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 15
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425&bid=18235470)

File: css/fonts.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+@font-face {
+font-family: 'Gotham Pro Regular';
+src: url('../fonts/GothamPro.eot');
```
Comment (KsuBurn) [inline css/fonts.css:3]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 2]
```diff
@@ -0,0 +1,19 @@
+@font-face {
+font-family: 'Gotham Pro Regular';
```
Comment (KsuBurn) [inline css/fonts.css:2]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+@font-face {
+font-family: 'Gotham Pro Bold';
+src: url('../fonts/GothamPro-Bold.eot');
+src: url('../fonts/GothamPro-Bold.eot?#iefix') format('embedded-opentype'),
+url('../fonts/GothamPro-Bold.woff') format('woff'),
+url('../fonts/GothamPro-Bold.ttf') format('truetype');
+font-weight: normal;
+font-style: normal;
+}
```
Comment (KsuBurn) [inline css/fonts.css:19]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+url('../fonts/GothamPro.ttf') format('truetype');
+font-weight: normal;
+font-style: normal;
+}
+
+@font-face {
+font-family: 'Gotham Pro Bold';
+src: url('../fonts/GothamPro-Bold.eot');
+src: url('../fonts/GothamPro-Bold.eot?#iefix') format('embedded-opentype'),
+url('../fonts/GothamPro-Bold.woff') format('woff'),
```
Comment (KsuBurn) [inline css/fonts.css:15]: Codacy Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+src: url('../fonts/GothamPro.eot');
+src: url('../fonts/GothamPro.eot?#iefix') format('embedded-opentype'),
+url('../fonts/GothamPro.woff') format('woff'),
+url('../fonts/GothamPro.ttf') format('truetype');
+font-weight: normal;
+font-style: normal;
+}
+
+@font-face {
+font-family: 'Gotham Pro Bold';
```
Comment (KsuBurn) [inline css/fonts.css:12]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 6]
```diff
@@ -0,0 +1,19 @@
+@font-face {
+font-family: 'Gotham Pro Regular';
+src: url('../fonts/GothamPro.eot');
+src: url('../fonts/GothamPro.eot?#iefix') format('embedded-opentype'),
```
Comment (KsuBurn) [inline css/fonts.css:4]: Codacy Issue found: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+url('../fonts/GothamPro.woff') format('woff'),
+url('../fonts/GothamPro.ttf') format('truetype');
+font-weight: normal;
+font-style: normal;
+}
+
+@font-face {
+font-family: 'Gotham Pro Bold';
+src: url('../fonts/GothamPro-Bold.eot');
```
Comment (KsuBurn) [inline css/fonts.css:13]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css (hunk 6 above)
Comment (KsuBurn) [inline css/fonts.css:4]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+src: url('../fonts/GothamPro-Bold.eot?#iefix') format('embedded-opentype'),
```
Comment (KsuBurn) [inline css/fonts.css:14]: Codacy Issue found: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

Comment (KsuBurn) [inline css/fonts.css:14]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 9]
```diff
@@ -0,0 +1,19 @@
+@font-face {
+font-family: 'Gotham Pro Regular';
+src: url('../fonts/GothamPro.eot');
+src: url('../fonts/GothamPro.eot?#iefix') format('embedded-opentype'),
+url('../fonts/GothamPro.woff') format('woff'),
+url('../fonts/GothamPro.ttf') format('truetype');
+font-weight: normal;
```
Comment (KsuBurn) [inline css/fonts.css:7]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 10]
```diff
@@ -0,0 +1,19 @@
+@font-face {
+font-family: 'Gotham Pro Regular';
+src: url('../fonts/GothamPro.eot');
+src: url('../fonts/GothamPro.eot?#iefix') format('embedded-opentype'),
+url('../fonts/GothamPro.woff') format('woff'),
```
Comment (KsuBurn) [inline css/fonts.css:5]: Codacy Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 11]
```diff
@@ -0,0 +1,19 @@
...
+font-weight: normal;
+font-style: normal;
+}
+
+@font-face {
+font-family: 'Gotham Pro Bold';
+src: url('../fonts/GothamPro-Bold.eot');
+src: url('../fonts/GothamPro-Bold.eot?#iefix') format('embedded-opentype'),
+url('../fonts/GothamPro-Bold.woff') format('woff'),
+url('../fonts/GothamPro-Bold.ttf') format('truetype');
```
Comment (KsuBurn) [inline css/fonts.css:16]: Codacy Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 12]
```diff
@@ -0,0 +1,19 @@
+@font-face {
+font-family: 'Gotham Pro Regular';
+src: url('../fonts/GothamPro.eot');
+src: url('../fonts/GothamPro.eot?#iefix') format('embedded-opentype'),
+url('../fonts/GothamPro.woff') format('woff'),
+url('../fonts/GothamPro.ttf') format('truetype');
```
Comment (KsuBurn) [inline css/fonts.css:6]: Codacy Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 13]
```diff
@@ -0,0 +1,19 @@
...
+font-weight: normal;
+font-style: normal;
```
Comment (KsuBurn) [inline css/fonts.css:8]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

**Comment by KsuBurn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 6
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425&bid=18235470)

File: css/fonts.css [hunk 14]
```diff
@@ -0,0 +1,19 @@
...
+  url('../fonts/GothamPro.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Gotham Pro Bold';
+  src: url('../fonts/GothamPro-Bold.eot');
+  src: url('../fonts/GothamPro-Bold.eot?#iefix') format('embedded-opentype'),
+  url('../fonts/GothamPro-Bold.woff') format('woff'),
```
Comment (KsuBurn) [inline css/fonts.css:15]: Codacy Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 15]
```diff
@@ -0,0 +1,19 @@
+@font-face {
+  font-family: 'Gotham Pro Regular';
+  src: url('../fonts/GothamPro.eot');
+  src: url('../fonts/GothamPro.eot?#iefix') format('embedded-opentype'),
+  url('../fonts/GothamPro.woff') format('woff'),
+  url('../fonts/GothamPro.ttf') format('truetype');
```
Comment (KsuBurn) [inline css/fonts.css:6]: Codacy Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 16]
```diff
@@ -0,0 +1,19 @@
...
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Gotham Pro Bold';
+  src: url('../fonts/GothamPro-Bold.eot');
+  src: url('../fonts/GothamPro-Bold.eot?#iefix') format('embedded-opentype'),
```
Comment (KsuBurn) [inline css/fonts.css:14]: Codacy Issue found: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 17]
```diff
@@ -0,0 +1,19 @@
+@font-face {
+  font-family: 'Gotham Pro Regular';
+  src: url('../fonts/GothamPro.eot');
+  src: url('../fonts/GothamPro.eot?#iefix') format('embedded-opentype'),
+  url('../fonts/GothamPro.woff') format('woff'),
```
Comment (KsuBurn) [inline css/fonts.css:5]: Codacy Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 18]
```diff
@@ -0,0 +1,19 @@
+@font-face {
+  font-family: 'Gotham Pro Regular';
+  src: url('../fonts/GothamPro.eot');
+  src: url('../fonts/GothamPro.eot?#iefix') format('embedded-opentype'),
```
Comment (KsuBurn) [inline css/fonts.css:4]: Codacy Issue found: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 19]
```diff
@@ -0,0 +1,19 @@
...
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Gotham Pro Bold';
+  src: url('../fonts/GothamPro-Bold.eot');
+  src: url('../fonts/GothamPro-Bold.eot?#iefix') format('embedded-opentype'),
+  url('../fonts/GothamPro-Bold.woff') format('woff'),
+  url('../fonts/GothamPro-Bold.ttf') format('truetype');
```
Comment (KsuBurn) [inline css/fonts.css:16]: Codacy Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

**Comment by KsuBurn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425&bid=18235470)

File: css/fonts.css (hunk 18 above)
Comment (KsuBurn) [inline css/fonts.css:4]: Codacy Issue found: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 20]
```diff
@@ -0,0 +1,19 @@
...
+    url('../fonts/GothamPro.woff') format('woff'),
+    url('../fonts/GothamPro.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Gotham Pro Bold';
+  src: url('../fonts/GothamPro-Bold.eot');
+  src: url('../fonts/GothamPro-Bold.eot?#iefix') format('embedded-opentype'),
```
Comment (KsuBurn) [inline css/fonts.css:14]: Codacy Issue found: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

**Comment by KsuBurn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425&bid=18235470)

File: css/fonts.css [hunk 21]
```diff
@@ -0,0 +1,21 @@
+@font-face {
+  font-family: 'Gotham Pro Regular';
+  src: url('../fonts/GothamPro.eot');
+  src:
```
Comment (KsuBurn) [inline css/fonts.css:4]: Codacy Issue found: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

Comment (KsuBurn) [inline css/fonts.css:4]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/fonts.css [hunk 22]
```diff
@@ -0,0 +1,21 @@
...
+    url('../fonts/GothamPro.eot?#iefix') format('embedded-opentype'),
```
Comment (KsuBurn) [inline css/fonts.css]: а не достаточно просто eot подключить? без url('../fonts/GothamPro.eot?#iefix') format('embedded-opentype'),
вот так url('../fonts/GothamPro.eot) format('eot'),

Comment (SergeyN83) [reply css/fonts.css]: Буду знать как лучше сделать, так предлагают сайты со шрифтами вставлять в код. А вообще всё убрал, кроме woff и woff2

Comment (SergeyN83) [reply css/fonts.css]: Нашел - iefix, это правка, чтобы под IE8 работали шрифты.

File: css/style.css [hunk 23]
```diff
@@ -0,0 +1,287 @@
...
+
+.product-card::after {
+  display: block;
+  content: '';
+  position: absolute;
+  left: 215px;
+  bottom: 84px;
+  width: 1px;
+  height: 100px;
+  opacity: 0.2;
```
Comment (KsuBurn) [inline css/style.css:182]: Codacy Issue found: [Unexpected leading zero (number-leading-zero)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

**Comment by KsuBurn** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 34
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425&bid=18235470)

File: css/style.css [hunk 24]
```diff
@@ -0,0 +1,287 @@
...
+  background: #000000;
```
Comment (KsuBurn) [inline css/style.css:183]: Codacy Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 25]
```diff
@@ -0,0 +1,287 @@
...
+.search-form__input {
+    height: 20px;
+    width: 185px;
+    border-bottom: 1px solid #000000;
+    background-color: #ffffff;
+    display: flex;
+    margin-right: 35px;
+}
+.search-form__text {
+  font-family: Gotham Pro Regular;
```
Comment (KsuBurn) [inline css/style.css:52]: Codacy Issue found: [Unexpected missing generic font family (font-family-no-missing-generic-family-keyword)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 26]
```diff
@@ -0,0 +1,287 @@
...
+  position: absolute;
+  left: 215px;
+  bottom: 84px;
+  width: 1px;
+  height: 100px;
+  opacity: 0.2;
+  background: #000000;
+}
+
+.product-card:last-child::after{
```
Comment (KsuBurn) [inline css/style.css:186]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 27]
```diff
@@ -0,0 +1,287 @@
...
+  padding-left: 60px;
+  max-width: 195px;
+}
+
+.main__header {
+  font-family: Gotham Pro Bold;
+  font-size: 30px;
+  line-height: 29px;
+  text-transform: uppercase;
+  color: #FFFFFF;
```
Comment (KsuBurn) [inline css/style.css:128]: Codacy Issue found: [Expected "#FFFFFF" to be "#ffffff" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 28]
```diff
@@ -0,0 +1,287 @@
...
+.product-card__btn_red {
+  background: #b62e3c;
+}
+
+.product-card__btn_yellow {
+  background: #fdb431;
+}
+
+.product-card__btn_brown {
+  background: #762E31;
```
Comment (KsuBurn) [inline css/style.css:252]: Codacy Issue found: [Expected "#762E31" to be "#762e31" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 29]
```diff
@@ -0,0 +1,287 @@
...
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  color: #000000;
+  margin-right: 35px;
+}
+
+.search-form__input {
+    height: 20px;
+    width: 185px;
```
Comment (KsuBurn) [inline css/style.css:45]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 30]
```diff
@@ -0,0 +1,287 @@
...
+.product-card__btn_brown {
+  background: #762E31;
+}
+
+.product-card__btn_orange {
+  background: #e29d69;
+}
+
+.product-card__btn_black {
+  background: #2c2c2c;
```
Comment (KsuBurn) [inline css/style.css:260]: Codacy Issue found: [Expected newline before "}" of a multi-line block (block-closing-brace-newline-before)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 31]
```diff
@@ -0,0 +1,287 @@
...
+  }
+
+  .container {
+    max-width: 930px;
+  }
+
+  .heading__logo {
+    margin-right: 29px;
+  }
+  .search-form__input {
```
Comment (KsuBurn) [inline css/style.css:277]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 32]
```diff
@@ -0,0 +1,287 @@
...
+  margin-right: 35px;
+}
+
+.search-form__input {
+    height: 20px;
+    width: 185px;
+    border-bottom: 1px solid #000000;
+    background-color: #ffffff;
+    display: flex;
+    margin-right: 35px;
```
Comment (KsuBurn) [inline css/style.css:49]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 33]
```diff
@@ -0,0 +1,287 @@
...
+
+.product-card__btn_black {
+  background: #2c2c2c;  
+}
+
+/*Медиа запросы*/
+
+@media (max-width: 1024px) {
+  body {
+  min-width: 1024px;
```
Comment (KsuBurn) [inline css/style.css:267]: Codacy Issue found: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 34]
```diff
@@ -0,0 +1,287 @@
...
+  line-height: 11px;
+  text-transform: uppercase;
+  color: #000000;
+  margin-right: 35px;
+}
+
+.search-form__input {
+    height: 20px;
+    width: 185px;
+    border-bottom: 1px solid #000000;
```
Comment (KsuBurn) [inline css/style.css:46]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 35]
```diff
@@ -0,0 +1,287 @@
...
+    background-color: #ffffff;
+    display: flex;
+    margin-right: 35px;
+}
+.search-form__text {
+  font-family: Gotham Pro Regular;
+  border: none;
+  outline: none; 
+  -webkit-appearance: none;
```
Comment (KsuBurn) [inline css/style.css:55]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 36]
```diff
@@ -0,0 +1,287 @@
...
+body {
+  min-width: 1170px;
+}
+
+.container {
+  margin: 0 auto;
+  max-width: 1030px;
+}
+
+/*Первый блок*/
```
Comment (KsuBurn) [inline css/style.css:15]: Codacy Issue found: [Expected whitespace after "/*" (comment-whitespace-inside)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 37]
```diff
@@ -0,0 +1,287 @@
...
+
+.heading__cart-img {
+  height: 15px;
+}
+
+.heading__flag {
+  width: 20px;
+}
+
+/*Второй блок*/
```
Comment (KsuBurn) [inline css/style.css:109]: Codacy Issue found: [Expected whitespace after "/*" (comment-whitespace-inside)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 38]
```diff
@@ -0,0 +1,287 @@
...
+  margin-right: 45px;
+}
+
+.navigation__link {
+  text-decoration: none;
+  font-family: Gotham Pro Bold;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  color: #000000;
```
Comment (KsuBurn) [inline css/style.css:39]: Codacy Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 39]
```diff
@@ -0,0 +1,287 @@
...
+
+/*Второй блок*/
+
+.main__wrapper {
+  height: 280px;
+  background: url(../img/banner-bg.png);
+  margin-bottom: 54px;
+}
+
+.main__text-wrapper{
```
Comment (KsuBurn) [inline css/style.css:117]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 40]
```diff
@@ -0,0 +1,287 @@
...
+    max-width: 930px;
+  }
+
+  .heading__logo {
+    margin-right: 29px;
+  }
+  .search-form__input {
+    margin-right: 32px;
+  }
+  .heading__signin {
```
Comment (KsuBurn) [inline css/style.css:280]: Codacy Issue found: [Expected empty line before rule (rule-empty-line-before)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 41]
```diff
@@ -0,0 +1,287 @@
...
+  font-family: Gotham Pro Bold;
+  font-size: 22px;
+  line-height: 21px;
+  text-transform: uppercase;
+  color: #000000;
+  text-align: center;
+  margin-bottom: 15px;
+}
+
+.main__card-wrapper{
```
Comment (KsuBurn) [inline css/style.css:163]: Codacy Issue found: [Expected single space before "{" (block-opening-brace-space-before)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css (hunk 34 above)
Comment (KsuBurn) [inline css/style.css:46]: Codacy Issue found: [Expected "#000000" to be "#000" (color-hex-length)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 42]
```diff
@@ -0,0 +1,287 @@
...
+  height: 9px;
+  width: 9px;
+}
+
+.product-card__btn_grey {
+  background: #c4c4c4;
+}
+
+.product-card__btn_blue {
+  background: #41479B;
```
Comment (KsuBurn) [inline css/style.css:240]: Codacy Issue found: [Expected "#41479B" to be "#41479b" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 43]
```diff
@@ -0,0 +1,287 @@
...
+    margin-right: 32px;
+  }
+  .heading__signin {
+    margin-right: 25px;
+  }
+
+  .heading__cart-wrapper {
+    margin-right: 27px;
+  }
+}
```
Comment (KsuBurn) [inline css/style.css:287]: Codacy Issue found: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 44]
```diff
@@ -0,0 +1,287 @@
...
+  font-family: Gotham Pro Bold;
+  font-size: 12px;
+  line-height: 11px;
+  text-transform: uppercase;
+  color: #000000;
+  margin-right: 35px;
+}
+
+.search-form__input {
+    height: 20px;
```
Comment (KsuBurn) [inline css/style.css:44]: Codacy Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

File: css/style.css [hunk 45]
```diff
@@ -0,0 +1,287 @@
...
+    width: 185px;
+    border-bottom: 1px solid #000000;
+    background-color: #ffffff;
+    display: flex;
+    margin-right: 35px;
+}
+.search-form__text {
+  font-family: Gotham Pro Regular;
+  border: none;
+  outline: none;
```
Comment (KsuBurn) [inline css/style.css:54]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

Comment (KsuBurn) [inline css/style.css:54]: Codacy Issue found: [Expected newline after ";" in a multi-line declaration block (declaration-block-semicolon-newline-after)](https://app.codacy.com/gh/WebHeroSchool/Axionproj/pullRequest?prid=5560425)

**Pull request merged**